examples/vpc-scenario-1/README.org

```diff
@@ -0,0 +1,40 @@
+** Example to test the VPC Scenario 1 Module
+
+*** Environment creation and deployment
+
+To use this example set up AWS credentials and then run the commands in the 
+following order:
+
+#+BEGIN_SRC bash
+make init
+make ssh-key
+make plan-vpc
+make apply
+make plan-subnets
+make apply
+make plan-gateway
+make apply
+#+END_SRC bash
+
+*** Testing
+
+Get the public IP address of the vpc machine with the AWS console.
```

Comment:
Does "vpc machine" refer to the web instance? This terminology is a little unclear.